drivers/include/nrfx_nvmc.h

```diff
@@ -37,6 +37,7 @@
 #include <nrfx.h>
 #include <hal/nrf_nvmc.h>
 #include <hal/nrf_ficr.h>
+#include <mdk/nrf_erratas.h>
```

Comment:
`mdk/` is redundant here and it would be better to remove it to keep consistency with other inclusions of this and other MDK headers.

Reply:
done